server/text_generation_server/models/__init__.py

```diff
@@ -28,7 +30,7 @@
 from text_generation_server.models.custom_modeling.bloom_modeling import (
     BloomForCausalLM,
 )
-from text_generation_server.models.globals import ATTENTION
+from text_generation_server.models.globals import ATTENTION, USE_CUSTOM_MODELING
```

Comment:
Remove this global flag. We already have `FLASH_ATTENTION` which serves the same purpose.

We do not want to add flags as much a possible.